### DISCUSSION TITLE
How to use .env file from Turborepo configs package

### DISCUSSION THREAD

**Discussion by OxyTJ** [prisma/prisma#18675] [Q&A] [answered]
I have a Turborepo setup where Prisma is being used in an `app` while I have my `.env` file in a `package`:

### Directory Structure:

- root
>- apps
>>- api
>>>- prisma
>>>>- schema.prisma
>- packages
>>- configs
>>>- .env

Looking at [Environment variables](https://www.prisma.io/docs/guides/development-environment/environment-variables#using-env-files), I am not seeing a way that this is possible, but wanting to confirm just in case since moving `.env` outside of the package completely breaks my flow and how the rest of the app is pulling various configurations besides just this `.env`. I've tried doing something like `npx prisma db push -- --dotenv=../../packages/configs/.env` to no avail - including using an absolute path.

The workaround would just be creating a separate `.env` file in root, solely being used by Prisma, which seems ridiculous.

**Comment by nurul3101** [comment, marked as answer]
Hello @OxyTJ 👋 

I was able to find this related GitHub Issue:

- https://github.com/prisma/prisma/issues/12535

Would using symlink solve the issue for you (as described in this [comment](https://github.com/prisma/prisma/issues/12535#issuecomment-1201808143))?

**Reply by OxyTJ** [reply]
Funny you mention that because I currently am using a symlink, however, I'd like to move away from that.

**Reply by nurul3101** [reply]
Ah, Okay, I am not aware of any other solutions that might help apart from Symlink. In case you find out another solution, please do share.
